jenkins/ci.fedora.Dockerfile

```diff
@@ -30,7 +30,8 @@
 FROM quay.io/fedora/fedora:34
 
 RUN \
-    dnf install -y \
+    dnf -y update && \
+    dnf install -y --exclude container-selinux \
```

Comment:
Why is it necessary to explicitly exclude `container-selinux`? That implies it was an indirect dependency but isn't actually *required* to complete the installs? Odd.

Reply:
This change is one I picked up on the way to [another solution](https://www.redhat.com/sysadmin/podman-inside-container) (which didn't pan out).  I figured the `update` was worth keeping (and there was a `reinstall` step which I dispensed with) and I didn't bother to drop the `container-selinux` exclusion because it seemed like having it was better than not having it.

The blog post says:
> We don’t install `container-selinux` because it is not needed within the container.

I suspect that it has no effect at all, but if that's so then it doesn't hurt anything.

Reply:
"A difference that makes no difference is no difference"

Reply:
That statement is only true during the interval in which the difference makes no difference; if, at some point in the future, the difference might actually make a difference, then, in the present, it is not no difference.  🙂

That is, I'm not sure whether we will stick with the `--security-opt label=disable` on the invocation, but, regardless, I believe that we don't really want or need SELinux to be active inside the container, so excluding it during the installations is probably a Good Thing.  (But, rather than trying to figure out how to actually remove/disable it, I think I'll move on to something more immediately useful....)